packages/env/src/index.ts

```diff
@@ -70,6 +71,7 @@ export const env = createEnv({
     FROM_MAILER_EMAIL: z.string(),
     GATEWAY_HOSTNAME: z.string(),
     LATITUDE_DOMAIN: z.string(),
+    LATITUDE_EMAIL_DOMAIN: z.string(),
```

Comment:
if this is not optional and you don't set them in web/gateway the production containers will fail

Reply:
Changed